torchgeo/datamodules/l7irish.py

```diff
@@ -68,9 +68,11 @@ def setup(self, stage: str) -> None:
         """
         dataset = L7Irish(**self.kwargs)
         generator = torch.Generator().manual_seed(0)
-        (self.train_dataset, self.val_dataset, self.test_dataset) = (
-            random_bbox_assignment(dataset, [0.6, 0.2, 0.2], generator)
-        )
+        (
+            self.train_dataset,
+            self.val_dataset,
+            self.test_dataset,
+        ) = random_bbox_assignment(dataset, [0.6, 0.2, 0.2], generator)
```

Comment:
Make sure you're using the latest version of black, the formatting has changed in the 2024 version.